.github/workflows/actions_build.yml

```diff
@@ -42,20 +42,22 @@ jobs:
             min-java: 11
           - java: 19
             os: ubuntu-latest
+          - java: 21
+            os: ubuntu-latest
```

Comment:
We usually use non-LTS for the latest version. Let's remove Java 19 version.
```suggestion
          - java: 21
            os: ubuntu-latest
```

Reply:
Updated